src/components/Nav.css

```
@@ -2,23 +2,26 @@
   position: -webkit-sticky;
   position: sticky;
   top: 0;
+  left: 10%;
+  right: 10%;
   z-index: 10;
 }
 
 .nav__content {
-  height: 40px;
+  height: 50px;
   display: flex;
-  justify-content: center;
+  justify-content: left;
```

This value is invalid
```suggestion
  justify-content: flex-start;
```